.gitignore

```diff
@@ -1,4 +1,13 @@
-.DS_Store
+# System files
 .idea
+.DS_Store
+.env
 .svn
 .vscode
+
+# Client files
+node_modules
+
+# Built files
+vendor
```

Comment:
Do we need the vendor files here since this is meant to be a built version of the plugin?